### ISSUE TITLE
Terminal app overflows note data to Notebook pane when wrapping text

### ISSUE THREAD

**Issue by Daeraxa** [laurent22/joplin#6227]
## Environment

Joplin CLI Client                                                                                                                                                             joplin 2.7.0 (dev, linux)
(Also with 2.6.2 prod, linux)                                                                                                                                       

Linux Mint 20.3
Replicated in a variety of combinations of terminal emulators (and shells).
Seen within GNOME Terminal, kitty and even Windows Terminal (via SSH session to the linux machine)
Launched from both bash and fish

## Steps to reproduce

1. Create note with data long enough that it can be easily wrapped for a given window size
2. Resize terminal window so note data starts to wrap in the preview pane
3. Text note data can be seen intruding into the Notebook pane (in the picture below the notebooks are all named "book" with an incrementing number. The overflow can be seen on book 4 & 6 where the wrapped lines appear

![image](https://user-images.githubusercontent.com/58074586/156274477-b44c797b-7a6b-4907-8081-f1a18b637541.png)

This does not happen purely on window resizing, also shows when app is started in a terminal of that size.

## Describe what you expected to happen

Text should be wrapped entirely within the constraints of the preview pane - data should not overflow to the notebook pane

## Logfile

Log does not update on window resizing so no relevant information appears to be written

**Comment by laurent22** [comment]
Does it make a difference if you use a different font? I've noticed that when the text contains Chinese characters or certain special unicode characters, but never with plain ascii.

**Comment by Daeraxa** [comment]
No, font changes don't seem to make any difference. On Windows terminal I'm using Caskaydia Cove which is a nerd font with ligature support but it shows up no matter what font I pick (Lucida console, consolas). 
On Linux I'm using Cascadia code (non ligature) in Kitty but it also shows up in GNOME terminal using its default Monospace Regular font. Also replicated with Source Code Pro, Ubuntu mono regular and Freemono Regular.

**Comment by Daeraxa** [comment]
For sake of completeness it can also be seen on macOS with the default terminal and Mono Regular font.

**Comment by Alzamer** [comment]
I would be interested in taking this on. Can somebody assign me?

**Comment by davidchilin** [comment]
Same issue on ubuntu 20.04, joplin terminal 2.6.2. Resizing not necessary to cause wrong wrap. As soon as I select a note it will happen.
Does not happen on all long lines, seems to happen more when a word is split at edge of preview pane.
As I scroll down through notes, the clipped wrap lines will be accumulated in Notebook pane.

---
unrelated note, is there a way to toggle off a pane, similar to gui version F10/F11. I currently use layout.xxxxx.factor, but does not allow value of int 0.

**Comment by Daeraxa** [comment]
I don't think so, I'm posting a bunch of enhancement issues to here soon in relation to [this post](https://discourse.joplinapp.org/t/terminal-app-feedback-improvement-ideas/25156/) and resizing + configuration of the panes is part of it.
